apps/ex_wire/lib/ex_wire/packet/status.ex

```diff
@@ -136,12 +138,27 @@ defmodule ExWire.Packet.Status do
   this peer. E.g. do our network and protocol versions match?
 
   ## Examples
+      iex> ExWire.BridgeSyncMock.start_link(%{})
```

Comment:
yeah, definitely tests that require mocking shouldn't be in doctests. https://github.com/poanetwork/blockscout/issues/290 good read!